cmd/litefs-bench/main.go

```diff
@@ -24,6 +24,18 @@ var (
 	maxRowsPerIter = flag.Int("max-rows-per-iter", 1000, "maximum number of rows per iteration")
 )
 
+func init() {
+	// Register a test driver for persisting the WAL after DB.Close()
+	sql.Register("sqlite3-persist-wal", &sqlite3.SQLiteDriver{
+		ConnectHook: func(conn *sqlite3.SQLiteConn) error {
+			if err := conn.SetFileControlInt("main", sqlite3.SQLITE_FCNTL_PERSIST_WAL, 1); err != nil {
+				return fmt.Errorf("cannot set file control: %w", err)
+			}
+			return nil
+		},
+	})
+}
+
```

Comment:
Added a custom driver to persist WAL files after databases are closed. This helps test WAL recovery.